### DISCUSSION TITLE
Using Clarity Core form controls in Angular Forms

### DISCUSSION THREAD

**Discussion by koninos** [vmware/clarity#6153] [General]
## Describe the bug

Not really a bug, but I am trying to use the new core components in place of the older Angular components and I cannot find in docs how to migrate the existing component `clr-control-error ` with `clrIfError` directive and use the new `cds-control-message` with similar functionality

`
<clr-password-container>
   <label>Password</label>
   <input clrPassword formControlName="password" type="password" />
   <clr-control-error *clrIfError="'required'">Password Required</clr-control-error>
</clr-password-container>
`

`
<cds-password control-width="shrink">
<label>Password</label>
<input formControlName="password" type="password" />
<cds-control-message status="error">error message</cds-control-message>
</cds-password>
`

## Versions

**Clarity project:**
Clarity Core 5.3.0
Clarity Angular/UI 5.3.0

**Clarity version:**
5.3.0

**Framework:**
Angular

**Framework version:**
Angular 11.2.13

**Comment by coryrylan** [comment]
You can use the angular forms module API directly without the need of the `*clr` type directives. Here is an example https://stackblitz.com/edit/node-p8km7t?file=src%2Fapp%2Fapp.component.html
